src/RestfulServer.php

```diff
@@ -493,7 +493,7 @@ protected function postHandler($className, $id, $relation)
                 return $this->notFound();
             }
 
-            if (!$obj->stat('allowed_actions') || !in_array($relation, $obj->stat('allowed_actions'))) {
+            if (!$obj->get('allowed_actions') || !in_array($relation, $obj->get('allowed_actions'))) {
```

Comment:
This is calling e.g. `Page::get`, where it should be `Config::get` - that is to say $obj isn't a `Config` instance, as the code seems to expect. You'll need to update this to:

```diff
- $obj->get(
+ Config::inst()->get($className,
```